css-two-column-layout/styles.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+  font-family: 'Karla', sans-serif, monospace;
+  background-color: rgb(255, 255, 255);
+}
+
+.container {
+  width: 90%;
+}
+
+.row {
+  display: flex;
+}
+
+.row-bottom-margin {
+  margin-bottom: 40px;
+}
+
+.width-80 {
+  width: 80%;
+}
+
+.header {
+  width: 60%;
+  text-align: center;
+  margin-top: 40px;
+  margin-bottom: 50px;
+}
+
+.header h1 {
+  font-size: 4.4rem;
+  margin-bottom: 20px;
+  letter-spacing: 3px;
+}
+
+.header p {
+  width: 90%;
+  color: rgb(90, 90, 90);
+  font-size: 1.2rem;
+  line-height: 1.9rem;
+  font-weight: 600;
+}
+
+.column {
+  width: 40%;
+}
+
+.column-left {
+  display: flex;
+  margin-right: 30px;
+}
+
+.column-left h4 {
+  margin-right: 140px;
```

Comment:
What is the purpose of this? Is this to center it or push content away from it? There is likely a cleaner way to accomplish this.